### ISSUE TITLE
Fix Gemini tipping logs user out 

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#21591]
## Description

https://github.com/brave/internal/issues/860

**Comment by kjozwiak** [comment]
Above will require `1.37.103` or later for `1.37.x` verification.

**Comment by GeetaSarvadnya** [comment]
Verification is inprogress

```

Brave | 1.37.103 Chromium: 100.0.4896.46 (Official Build) beta (64-bit)
-- | --
Revision | 5ca33821b2211805855c77d334353d27c616a7ca-refs/branch-heads/4896@{#584}
OS | Windows 10 Version 21H2 (Build 19044.1586)

```

**Comment by LaurenWags** [comment]
Removing `OS/Android` as Gemini is not available on Android for 1.37.x
